.changelog/unreleased/enhancement/245-rename-ics-context.md

```diff
@@ -0,0 +1,2 @@
+- Rename Ics* names to something more descriptive
```

Comment:
Can we move this to `.changelog/unreleased/breaking-changes/` directory?

Reply:
done